src/shogun/labels/BinaryLabels.cpp

```diff
@@ -148,3 +149,53 @@ CLabels* CBinaryLabels::shallow_subset_copy()
 
 	return shallow_copy_labels;
 }
+
+namespace shogun
+{
+	Some<CBinaryLabels> binary_from_binary(CBinaryLabels* orig)
+	{
+		return Some<CBinaryLabels>::from_raw(orig);
+	}
+
+	Some<CBinaryLabels> binary_from_dense(CDenseLabels* orig)
```

Comment:
que?
do we need to spell this one out? does this need to be expose to anybody apart from ourself internally?

Reply:
ah this is in .cpp. nevermind

Reply:
no this is only internally, didnt put it in the header.
You think it should be just inside the `switch`?

Reply:
see the 2nd comment... it's foobar, just ignore the first comment

Reply:
browser didnt reload ;)